views/partials/navbar.ejs

```diff
@@ -3,12 +3,23 @@
 
   <div class="nav-links">
     <% if(!currentUser) {%>
-    <a href="/login">Login</a>
-    <a href="/register">Register</a>
+    <a class="nav-link-guest" href="/login">Login</a>
+    <a class="nav-link-guest" href="/register">Register</a>
     <% } else { %>
-    <a href="/play">Play</a>
-    <a href="/account">My Account</a>
-    <a href="/logout">Log out</a>
+    <a class="nav-link" href="/play">Play</a>
+    <a class="nav-link" href="/account">My Account</a>
+    <a class="nav-link" href="/logout">Log out</a>
+
+    <a class="icon-wrapper" href="/play"
+      ><img class="nav-icon" src="../icons/play_icon.png" alt="play"
+    /></a>
+    <a class="icon-wrapper" href="/account"
+      ><img class="nav-icon" src="../icons/account_icon.png" alt="account"
+    /></a>
+    <a class="icon-wrapper" href="/logout"
+      ><img class="nav-icon" src="../icons/logout_icon.png" alt="logout"
+    /></a>
+    <div class="icon-wrapper"></div>
```

Comment:
What is the purpose of this empty `div`?

Reply:
Is unintentional leftover code, will get rid of it